### DISCUSSION TITLE
Updating Elyra on JupyterHub leads to endpoints not found

### DISCUSSION THREAD

**Discussion by dkogut1996** [elyra-ai/elyra#1506] [Q&A] [answered]
I am running Elyra on JupyterHub (k8s-hub:0.11.1) + Enterprise-Gateway (2.5.0) on GKE. This is run successfully using Elyra 1.5.3, but when updating to 2.1.0, I get the following errors in succession:

- Error contacting server: Endpoint elyra/namespace not found.
- Error contacting server: Endpoint elyra/schema/code-snippets not found.
- Error contacting server: Endpoint elyra/schema/runtimes not found.
- Failed to load the jupyterlab-git server extension: Git server extension is unavailable. Please ensure you have installed the JupyterLab Git server extension by running: pip install --upgrade jupyterlab-git. To confirm that the server extension is installed, run: jupyter server extension list.

Is this known behavior/do you have any ideas for fixing this behavior?

**Comment by lresende** [comment, marked as answer]
@dkogut1996 This has been fixed as part of Elyra 2.2.0, see https://github.com/elyra-ai/elyra/issues/1474

**Reply by kevin-bates** [reply]
In the meantime (prior to 2.2.0's availability - which should be later today) you can use the `:dev` tag of the elyra image.

**Reply by dkogut1996** [reply]
Thank you!!

**Comment by ptitzler** [comment]
The 2.2.x release containing the fix is now available. Details are in the [announcement thread](https://github.com/elyra-ai/elyra/discussions/1260).
